core/traits/long_neck.py

```diff
@@ -8,13 +8,23 @@ def __init__(self):
         name = type(self).__name__
         categories = tuple([TraitCategory.EATING])
         super().__init__(name, categories)
-        dispatcher.connect(self.handler_start_feeding_phase, Signal.START_FEEDING_PHASE)
+        dispatcher.connect(
```

Comment:
Not 100% sure if this would work, but could we use a decorator to automatically handle the dispatcher connectivity? Rather than having to have lots of
`dispatcher.connect(self.handler,signal)`
calls scattered throughout the code we could do something like this:
`@connect(Signal.START_FEEDING_PHASE)
def handler_foo():
    # Do whatever
`